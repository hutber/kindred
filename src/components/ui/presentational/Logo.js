//- Created by hutber on 22/04/16.  */
import React from 'react';
import logo from '../../../../css/core/_Logo.css';
import * as font from '../../../../css/font/fontello.css';

const Logo = React.createClass ({
    render() {
        return (
            <div className={logo.logo +' '+font['icon-logo']}></div>
        );
    }
});

export default Logo;
