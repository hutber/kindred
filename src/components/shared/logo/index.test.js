import React from 'react';
import Logo from './index';
import ShallowRenderer from 'react-test-renderer/shallow'

describe('</Splash>', () => {
	it('Render the logo', () => {
		const renderer = new ShallowRenderer();
		const result = renderer.render(<Logo />);
		expect(result).toMatchSnapshot();
	});
});
