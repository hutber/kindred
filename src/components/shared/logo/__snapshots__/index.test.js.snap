// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</Splash> Render the logo 1`] = `
<div
  className="logo undefined"
/>
`;
